README.md

```diff
@@ -1 +1,141 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Introduction
+
+The parliament challenge by Trustly is approached as follows:
+
+1. Get speeches resource from its respective apis. Since speeches data is in a nested dict, the for loop has been used to iterate over the required dict which is "anforande". The speeches api allows for extracting the ten latest speeches.
```

Comment:
Is it really nested? How?

Reply:
the resource is as follows:

{
  "anforandelista": {
    "@antal": "10",
    "anforande": [
      {"keys": "values",
       "keys": "values"
}]

so it becomes nested